tools/apigee-envoy-quickstart/README.md

```diff
@@ -79,7 +79,7 @@ The Apigee Envoy Quickstart Toolkit sets up the Envoy proxies with Apigee adapte
 ### Cleanup
 1. **Run to cleanup the PoC setup from the GKE and Apigee platform**
     ```bash
-    cd ${ENVOY_HOME}/apigee-envoy-quickstart-main
+    cd ${ENVOY_HOME}/apigee-devrel/tools/apigee-envoy-quickstart/
```

Comment:
I'm a little confused by this. Is the entire devrel repo cloned inside of `$ENVOY_HOME`?

Reply:
Yes. entire devrel is cloned and executed from within apigee-envoy-quickstart directory.